### PR TITLE
bpo-36999: Add asyncio.Task.get_coro()

### PR DESCRIPTION
**Pull request by agronholm**
https://bugs.python.org/issue36999

### CODE REVIEW

File: Doc/library/asyncio-task.rst [hunk 1]
```diff
@@ -842,6 +842,12 @@ Task Object
       The *file* argument is an I/O stream to which the output
       is written; by default output is written to :data:`sys.stderr`.
 
+   .. method:: get_coro()
+
+      Return the coroutine object that was passed to the constructor.
```
Comment (asvetlov) [inline Doc/library/asyncio-task.rst]: I dislike mentioning `constructor` here.

Above we have the text:
```
   Use the high-level :func:`asyncio.create_task` function to create
   Tasks, or the low-level :meth:`loop.create_task` or
   :func:`ensure_future` functions.  Manual instantiation of Tasks
   is discouraged.
```
Maybe rephrase this line to `Return the coroutine object that was used for the task creation.` or something like this?
Sorry, I'm not a native speaker. Don't insist on the exact wording but you've got my idea.

Comment (1st1) [reply Doc/library/asyncio-task.rst]: I'd just say this: "Return the coroutine object wrapped by the Task".

**Review by 1st1 — APPROVED** [review]

File: Doc/library/asyncio-task.rst (hunk 1 above)
Comment (asvetlov) [inline Doc/library/asyncio-task.rst]:
```suggestion
      Return the coroutine object wrapped by the :class:`Task`.
```
@1st1 like this?

Comment (1st1) [reply Doc/library/asyncio-task.rst]: Yeah, I think it's great.  If you have time just merge Alex's PR and we can fix this ourselves. Up to you.

Comment (agronholm) [reply Doc/library/asyncio-task.rst]: I changed the wording now as requested.

Comment (asvetlov) [reply Doc/library/asyncio-task.rst]: Great!

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
Thanks!
